jenkins/docker/Jenkinsfile

```diff
@@ -27,9 +29,19 @@ pipeline {
             name: 'IS_STAGING',
             description: 'Are we pushing docker images to staging (opensearchstaging) or production (opensearchproject) account.'
         )
+        string(
```

Comment:
Looks like these parameters are derived from another parameter, lets move this logic into code rather than force the caller to know which field to populate

Reply:
Updated the parameters